<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SshServerOptionsWidget</class>
 <widget class="QWidget" name="SshServerOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="0" column="0" rowspan="3">
    <widget class="QListWidget" name="serversList"/>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="configContainer" native="true">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>370</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>370</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="addNewButton">
     <property name="text">
      <string>Add New...</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="deleteButton">
     <property name="text">
      <string>Delete Selected</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
